server/modules/form/middleware.js

```diff
@@ -16,14 +16,34 @@ export default {
 	},
 
 	setLocals: (req, res, next) => {
-		res.locals.marketingName = req.query.marketingName && escape(req.query.marketingName) || FALLBACK_MARKETING_NAME;
+		res.locals.marketingName = escape(req.params.marketingName || req.query.marketingName || '') || FALLBACK_MARKETING_NAME;
```

Comment:
Instead of `''` you could put the `FALLBACK_MARKETING_NAME` in the escape?

Reply:
I had that originally but then it escapes the spaces and changes the value. Just worried that something somewhere is directing leads based off "propensity organic form" and wont work for "propensity%20organic%20form"